cmd/root.go

```diff
@@ -114,6 +120,10 @@ any client SSL certificates.`,
 		"Path to a service account key to use for authentication.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.GcloudAuth, "gcloud-auth", "g", false,
 		"Use gcloud's user configuration to retrieve a token for authentication.")
+	cmd.PersistentFlags().StringVar(&c.prometheusNamespace, "prometheus-namespace", "",
+		"Enable Prometheus for metric collection using the provided namespace")
+	cmd.PersistentFlags().StringVar(&c.prometheusPort, "prometheus-port", "9090",
```

Comment:
I think we chatted about this and decided we only wanted want HTTP server, even for things like health checks and this endpoint. 

If so, we probably want to remove this flag to something like "--http-port" or "--http-server-port"

We also probably want to abstract it out and register endpoints as needed, but maybe that can go in a different PR.

Reply:
Sounds good. I'll update here.

Reply:
I agree -- let's extract the HTTP server on the next flag that needs its own. I'll handle the endpoint registration in that future PR.